src/services/ui/src/components/Container/SimplePageContainer.tsx

```diff
@@ -0,0 +1,5 @@
+import { PropsWithChildren } from "react";
```

Comment:
Good call on this component. We were definitely duplicating some code there before